Set ifort configuration

erothe opened the pull request:
Add the following line to `ifort.cfg`:
```
-isystem/ssoft/spack/syrah/v1/opt/spack/linux-rhel8-x86_64_v2/gcc-8.5.0/intel-oneapi-compilers-classic-2021.6.0-q3mi2mylw3zyuht6p72u25ruqnpptpym/compiler/include/intel64
```
ifort.cfg is located in

Commit:
Set ifort configuration

nrichart commented on jenkins/deploy/scripts/install_compilers_parallel.sh:
```diff
@@ -46,6 +46,17 @@ do
+# Hack for intel compiler.
+# We want to add the following line to ifort.cfg (overwrite).
+# -isystem/ssoft/spack/syrah/v1/opt/spack/linux-rhel8-x86_64_v2/gcc-8.5.0/intel-oneapi-compilers-classic-2021.6.0-q3mi2mylw3zyuht6p72u25ruqnpptpym/compiler/include/intel64
+DST_PRFX=`spack location -i intel-oneapi-compilers`
```
why not using `$(spack location -i intel-oneapi-compilers-classic)/bin/ifort.cfg`
has the destination ?

erothe replied:
You're right, I've got the wrong path. ```DST_PRFX=`spack location -i intel-oneapi-compilers`/bin``` is fine.

erothe replied:
I have added a new commit following your comment.

Commit:
Update install_compilers_parallel.sh

I have updated the file according to your comments.

nrichart commented:
We also need to modify the config for icc/icpc with
```
-isystem/ssoft/spack/syrah/v1/opt/spack/linux-rhel8-x86_64_v2/gcc-8.5.0/intel-oneapi-compilers-classic-2021.6.0-q3mi2mylw3zyuht6p72u25ruqnpptpym/compiler/include/icc
```
for `immintrin.h`  #23 was not correct

erothe reviewed (commented):
@nrichart Can you approve ?

erothe reviewed (commented):
OK

Commit:
Mark as system directories

Set configuration to point to intel-oneapi-compilers-classic (ifort.cfg, icc.cfg and icpc.cfg)

erothe commented:
Changes made, ready to review and approve.

nrichart reviewed (approved):
LGTM
